src/main.lua

```diff
@@ -409,6 +408,17 @@ local function updateLastKnownMessage(msg)
 	end
 end
 
+--- @class ParsedMessage
+--- @field Id string
+--- @field From string
+--- @field Timestamp number
+--- @field Tags table<string, any>
+
+--- @param handlerName string
+--- @param pattern fun():string
+--- @param handleFn fun(msg: ParsedMessage)
+--- @param critical boolean?
+--- @param printEvent boolean?
```

Comment:
gives us IDE types in main.handlers
<img width="557" alt="image" src="https://github.com/user-attachments/assets/431bf77d-4c4a-40cf-beac-606654e68e1b" />

Reply:
will include ioEvent